sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracerProvider.java

```diff
@@ -100,6 +102,17 @@ public Sampler getSampler() {
     return sharedState.getSampler();
   }
 
+  // currently not public as experimental
```

Comment:
The pattern for giving access to these experimental features has been via `Sdk{Signal}ProviderUtil`, a public class in an internal package, with static helper methods for reflectively accessing package private methods.

For tracing, this is `SdkTracerProviderUtil`.

I'd recommend adding an accessor method there as part of this PR. If you do, please add javadoc to this pointing to that method.

If you choose to punt on that, let's add a TODO to followup:
```suggestion
  // TODO: add public experimental accessor in SdkTracerProviderUtil
```

Speaking of followups, after landing this, should open an issue to track extending metrics and logs with the same capability.

Reply:
Added the static helper, and opened https://github.com/open-telemetry/opentelemetry-java/issues/7038 for extending to metrics and logs